packages/@aws-cdk/aws-servicecatalog/README.md

```diff
@@ -202,15 +202,24 @@ portfolio.addProduct(product);
 
 TagOptions allow administrators to easily manage tags on provisioned products by creating a selection of tags for end users to choose from.
 For example, an end user can choose an `ec2` for the instance type size.
-TagOptions are created by specifying a key with a selection of values.
+TagOptions are created by specifying a key with a selection of values, 
+and can be associated with both portfolios and products.  When launching a product,
+both the TagOptions associated with the product and the containing portfolio are made available.
+
 At the moment, TagOptions can only be disabled in the console.
 
-```ts fixture=basic-portfolio
-const tagOptions = new servicecatalog.TagOptions({
+```ts fixture=portfolio-product
+const tagOptionsForPortfolio = new servicecatalog.TagOptions({
+  costCenter: ['A', 'B'],
```

Comment:
Again, can we make it a real example when would you use these? Not just some dummy values?

Reply:
yeah I am not fully aware of how these are usually done since mostly they are like account numbers.  I referenced our own public [docs](https://docs.aws.amazon.com/servicecatalog/latest/adminguide/tagoption-policies.html) and used that example.